WalletWasabi.Fluent/CrashReport/ViewModels/CrashReportWindowViewModel.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.IO;
+using ReactiveUI;
+using System.Windows.Input;
+using WalletWasabi.Fluent.ViewModels.HelpAndSupport;
+using WalletWasabi.Models;
+using WalletWasabi.Gui.ViewModels;
+
+namespace WalletWasabi.Fluent.CrashReport.ViewModels
+{
+	public partial class CrashReportWindowViewModel : ViewModelBase
+	{
+		[AutoNotify] private SerializableException _serializableException;
```

Comment:
I can't see why is it a property or even a `[AutoNotify]` one. I would change it to a readonly field and the `partial` keyword can be removed too from next to class.